### DISCUSSION TITLE
Bunny Patrol

### DISCUSSION THREAD

**Discussion by djcsdy** [excaliburjs/Excalibur#1845] [Show and tell]
Hi!

A friend (@Zutty) and I have been converting some old Flash games to Excalibur. I can't remember if I've submitted this one to Show and Tell before, so you might already have seen it.

# [Bunny Patrol](https://bunny-patrol.vidja.games/)
[![image](https://user-images.githubusercontent.com/310800/114108104-46964100-98ca-11eb-9756-f6ca04fd8fb6.png)](https://bunny-patrol.vidja.games/)

George Weller originally made this in Flash for Glorious Trainwrecks Pirate Kart V (the 2012-in-one Glorious Developers Konference Kollection). I helped a little bit with sound and also did some of the conversion work, but this game is 99% George's work.

 * [Bunny Patrol](https://bunny-patrol.vidja.games/) - the game itself
 * [Description page on Glorious Trainwrecks](https://www.glorioustrainwrecks.com/node/2654)
 * [Source Code](https://github.com/dcgw/bunny-patrol)

# [You Are Now In Space](https://you-are-now-in-space.vidja.games/)

[![image](https://user-images.githubusercontent.com/310800/114108471-0c796f00-98cb-11eb-90ae-fae00f3fd102.png)](https://you-are-now-in-space.vidja.games/)

I've submitted this before but here it is again for anyone who missed it. George and I made this in Flash in 2011 for Mini Ludum Dare 25. The theme of the competition was "The Worst Game I Have Ever Made" so the game is... intentionally bad.

 * [You Are Now In Space](https://you-are-now-in-space.vidja.games/) - the game itself
 * [Description page on my personal website](https://www.danielcassidy.me.uk/games/you-are-now-in-space/)
 * [Source Code](https://github.com/dcgw/you-are-now-in-space)

---

You can see the other stuff we're working on at https://github.com/dcgw

**Comment by kamranayub** [comment]
Ahhhh this makes my month 🎉 And thanks for providing all your feedback to help make the engine easier to use!
